docs/release-info/release-notes.md

```diff
@@ -73,47 +132,208 @@ This section contains detailed release notes separated by areas.
+- `ifSynced`: Reads are done from the cache only if it has already synced with the metadata store. This mode does not block service start-up or transactions unlike the `always` setting.
+
+As part of this change, additional metrics have been introduced. For more information about these metrics, see [Segment metadata cache metrics](#segment-metadata-cache-metrics).
+
+[#17653](https://github.com/apache/druid/pull/17653) [#17824](https://github.com/apache/druid/pull/17824)
+
+#### Automatic kill task interval
+
+The Coordinator can optionally issue kill tasks for cleaning up unused segments. Starting with this release, individual kill tasks are limited to processing 30 days or fewer worth of segments per task by default. This improves performance of the individual kill tasks.
+
+The previous behavior (no limit on interval per kill task) can be restored by setting `druid.coordinator.kill.maxInterval = P0D.`
+
+[#17680](https://github.com/apache/druid/pull/17680)
+
 #### Other data management improvements
 
+- Metadata queries now return `maxIngestedEventTime`, which is the timestamp of the latest ingested event for the datasource. For realtime datasources, this may be later than `MAX(__time)` if `queryGranularity` is being used. For non-realtime datasources, this is equivalent to `MAX(__time)` [#17686](https://github.com/apache/druid/pull/17686)
+- Metadata kill queries are now more efficient. They consider a maximum end time since the last segment was killed [#17770](https://github.com/apache/druid/pull/17770)
+- Newly added segments are loaded more quickly [#17732](https://github.com/apache/druid/pull/17732)
+
 ### Metrics and monitoring
 
+#### Custom Histogram buckets for Prometheus
+
+You can now configure custom Histogram buckets for `timer` metrics from the Prometheus emitter using the `histogramBuckets` parameter. 
+
+If no custom buckets are provided, the following default buckets are used: `[0.1, 0.25, 0.5, 0.75, 1.0, 2.5, 5.0, 7.5, 10.0, 30.0, 60.0, 120.0, 300.0]`. If the user does not specify their own JSON file, a default mapping is used.
+
+[#17689](https://github.com/apache/druid/pull/17689)
+
+#### Segment metadata cache metrics
+
+The following metrics have been added:
+
+The following metrics have been introduced as part of the segment metadata cache performance improvement.
+
+- `segment/used/count`
+- `segment/unused/count`
+- `segment/pending/count`
+- `segment/metadataCache/sync/time`
+- `segment/metadataCache/deleted`
+- `segment/metadataCache/skipped`
+- `segment/metadataCache/used/stale`
+- `segment/metadataCache/used/updated`
+- `segment/metadataCache/unused/updated`
+- `segment/metadataCache/pending/deleted`
+- `segment/metadataCache/pending/updated`
+- `segment/metadataCache/pending/skipped`
```

Comment:
I think we should mention only the following metrics here.
Since this is an experimental feature, several other metrics have been added to track performance but may be removed in the near future.

- `segment/metadataCache/sync/time`
- `segment/metadataCache/transactions/readOnly`
- `segment/metadataCache/transactions/writeOnly`
- `segment/metadataCache/transactions/readWrite`